src/writer/launchdarkly_client.py

```
@@ -0,0 +1,246 @@
+        except Exception as e:
+            logger.warning(
+                f"LaunchDarkly bulk flag evaluation failed: {e}",
+                exc_info=True
+            )
+            return {}
+
+    @classmethod
+    def capture_error(
+        cls,
+        error: Exception,
+        context: Optional[Dict[str, Any]] = None,
```

Ruff (0.14.8)

Unused class method argument: `context`

(ARG003)